client/src/components/DirectoryAddUpdate.jsx

```diff
@@ -13,28 +13,30 @@ import Country from "./Country.jsx";
 import Province from "./Province.jsx";
 import { DEFAULT_COUNTRY_CODE } from "../utils/constants";
 
+import { useAlert } from "../components/AlertProvider";
+
 import "../containers/FormContainer.css";
 
 const { REACT_APP_PROXY } = process.env;
 
 export default function DirectoryAddUpdate() {
   const [coopName, setCoopName] = useState("");
   const [street, setStreet] = useState("");
-  const [addressPublic, setAddressPublic] = useState("no");
+  const [addressPublic, setAddressPublic] = useState("No");
   const [city, setCity] = useState("");
   const [state, setState] = useState("IL");
   const [zip, setZip] = useState("");
   const [county, setCounty] = useState("");
   const [country, setCountry] = useState("US");
   const [websites, setWebsites] = useState("");
   const [contactName, setContactName] = useState("");
-  const [contactNamePublic, setContactNamePublic] = useState("no");
+  const [contactNamePublic, setContactNamePublic] = useState("No");
```

Comment:
As we use the magic string "No", in several places, let's define it as a constant in the src/utils/constants.jsx file and reference it here.

Reply:
This is done! Let me know if that name works for the variable.